.gitignore

```diff
@@ -72,3 +72,5 @@ fastlane/test_output
 # The frameworks folder is where they should go, but keep the frameworks themselves out of Xcode
 Frameworks/*
 
+**/WatsonMLClientCredentials.json
```

Comment:
@sanjeevghimire make sure to share this file in the slack chatroom so we can all place it in the right place. @antonmc and @stevemart this will have to go in a specific place in the client application folder so calls to the server work fine.